packages/subgraph/tests/addresslist-voting/addresslist-voting.test.ts

```diff
@@ -47,24 +52,34 @@ import {
   getProposalCountCall,
   createAddresslistVotingProposalEntityState,
 } from './utils';
-import {Address, BigInt, Bytes} from '@graphprotocol/graph-ts';
+import {
+  generateEntityIdFromAddress,
+  generatePluginEntityId,
+} from '@aragon/osx-commons-subgraph';
+import {Address, BigInt, Bytes, log} from '@graphprotocol/graph-ts';
 import {assert, clearStore, test} from 'matchstick-as/assembly/index';
 
 let actions = createDummyActions(DAO_TOKEN_ADDRESS, '0', '0x00000000');
 
+const daoAddress = Address.fromString(DAO_ADDRESS);
+const daoEntityId = generatePluginEntityId(daoAddress);
+const pluginAddress = Address.fromString(CONTRACT_ADDRESS);
+const pluginEntityId = generatePluginEntityId(pluginAddress);
+const memberOneAddress = Address.fromString(ADDRESS_ONE);
+const memberTwoAddress = Address.fromString(ADDRESS_TWO);
+const memberOneId = generateEntityIdFromAddress(memberOneAddress);
```

Comment:
using `generateEntityIdFromAddress()` implies that the output will be used as an entity ID
but in reality you just want the hex string representation of this members address

also the naming of `memberOneId` seems off, it is not the addressList member (voter) id structure of (pluginId + '_' + memberAddressHexString)

the easiest suggestion here would be to use `memberOneAddress.toHexString()` and not have `memberOneId`, or you can come up with a better solution if you like.

if you agree please revisit other ocurrance of this type

@Michael-A-Heuer for visibility and second opinion

Reply:
Yes, sounds reasonable

Reply:
renamed them in [7e98a90](https://github.com/aragon/osx/pull/520/commits/7e98a90c608bf5e6dfaf7a66b02d887853de5407)